README.md

```diff
@@ -21,6 +21,52 @@ and build the library with default configuration values, provide `MQTT_DO_NOT_US
 **OR**
 * Defining the `MQTT_DO_NOT_USE_CUSTOM_CONFIG` preprocessor macro for the library build.
 
+## Sending metrics to AWS IoT
+
+When establishing a connection with AWS IoT, users can optionally report the Operating System, Hardware Platform and MQTT client version information of their device to AWS. These metrics help AWS IoT improve security and provide better technical support. If users want to report this information, they can send a specially formatted string (see below) in the username field of the MQTT CONNECT packet.
```

Comment:
Second sentence --> "This information can help AWS IoT provide faster issue resolution and technical support."

Reply:
Fixed

Reply:
Fixed